classes/class-db-driver-wpdb.php

```diff
@@ -0,0 +1,157 @@
+<?php
+namespace WP_Stream;
+
+class DB_Driver_WPDB implements DB_Driver {
+	/**
+	 * Hold Query class
+	 * @var Query
+	 */
+	protected $query;
+
+	/**
+	 * Hold records table name
+	 *
+	 * @var string
+	 */
+	public $table;
+
+	/**
+	 * Hold meta table name
+	 *
+	 * @var string
+	 */
+	public $table_meta;
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+		$this->query  = new Query( $this );
+
+		global $wpdb;
+		$prefix = apply_filters( 'wp_stream_db_tables_prefix', $wpdb->base_prefix );
+
+		$this->table         = $prefix . 'stream';
+		$this->table_meta    = $prefix . 'stream_meta';
+
+		$wpdb->stream        = $this->table;
+		$wpdb->streammeta    = $this->table_meta;
+
+		// Hack for get_metadata
+		$wpdb->recordmeta    = $this->table_meta;
```

Comment:
Too much spacing after each of these properties. They can be aligned with less spaces.